disasm-test/Main.hs

```diff
@@ -512,6 +652,23 @@ processBitCode pfx file = do
         when det $ liftIO $ putStrLn $ "## parsed Bitcode to " <> parsed
         return (parsed, Nothing)
 
+
+postParseTests :: AST.Module -> TestM ()
```

Comment:
This could use a comment explaining what it is checking for.

Reply:
Added.